build.sbt

```diff
@@ -0,0 +1,29 @@
+val tapirVersion = "1.1.0"
+
+lazy val rootProject = (project in file("."))
+  .enablePlugins(ScalaxbPlugin)
+  .settings(
+    Seq(
+      name := "tapir-scalaxb-example",
+      version := "0.1.0-SNAPSHOT",
+      organization := "com.softwaremill.tapir.scalaxb.example",
+      scalaVersion := "2.13.8",
+      libraryDependencies ++= Seq(
+        "com.softwaremill.sttp.tapir" %% "tapir-http4s-server" % tapirVersion,
+        "org.http4s" %% "http4s-blaze-server" % "0.23.12",
+        "com.softwaremill.sttp.tapir" %% "tapir-swagger-ui-bundle" % tapirVersion,
+        "com.softwaremill.sttp.tapir" %% "tapir-json-circe" % "1.1.0",
```

Comment:
Yes, we need to have proper namespace param in swagger's example requests, so that scalaxb code could decode them. It is done by adding doc atribute to `Schema` by this method from sttp.tapir.docs.apispec.DocsExtensionAttribute:

```
  implicit class RichSchema[T](s: Schema[T]) {
    def docsExtension[D: JsonCodec](key: String, value: D): Schema[T] =
      s.attribute(docsExtensionAttributeKey, docsExtensions :+ DocsExtension.of(key, value))
    def docsExtensions: Vector[DocsExtension[_]] = s.attribute(docsExtensionAttributeKey).getOrElse(Vector.empty)
  }
  ```
  
  and this metod needs `JsonCodec`, thus cice is used.

Reply:
ah ... all clear, thanks! :)

though ... maybe a short comment in build.sbt on why this is needed would be useful>